framework/include/constraints/ElemElemConstraint.h

```diff
@@ -0,0 +1,127 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#ifndef ELEMELEMCONSTRAINT_H
+#define ELEMELEMCONSTRAINT_H
+
+// MOOSE includes
+#include "Constraint.h"
+#include "NeighborCoupleableMooseVariableDependencyIntermediateInterface.h"
+#include "MooseMesh.h"
+#include "ElementPairInfo.h"
+
+// Forward Declarations
+class ElemElemConstraint;
+class FEProblem;
+
+template<>
+InputParameters validParams<ElemElemConstraint>();
+
+class ElemElemConstraint :
+  public Constraint,
+  public NeighborCoupleableMooseVariableDependencyIntermediateInterface
```

Comment:
Like this?
```
class ElemElemConstraint :
    public Constraint,
    public NeighborCoupleableMooseVariableDependencyIntermediateInterface
```
I'm happy to do this, but it doesn't say anything about that in the Code Standards, and the few random files that I looked at in the framework to find an example all use 2 spaces. If the accepted standard is to indent 4 spaces, could someone please update the Code Standards to include that?